src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/probationoffendersearch/Address.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.probationoffendersearch
+
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Address
+
+data class Address(
+  val postcode: String
+) {
+  fun toAddress(): Address = Address(postcode = this.postcode)
```

Comment:
You shouldn't need to explicitly specify the type here on toAddress(). Although we're doing it in other cases so I wouldn't bother changing it unless we do it for all, just a thought for refactoring in the future.

Reply:
Here it's super clear what the return type is so I've removed it, thanks for the suggestion!